com.unity.shadergraph/Editor/Generation/Enumerations/Platform.cs

```diff
@@ -63,5 +63,11 @@ internal static Platform[] GetHighEndPlatformArray()
         {
             return new Platform[] { Platform.D3D11, Platform.Playstation, Platform.XboxOne, Platform.GameCoreXboxSeries, Platform.Vulkan, Platform.Metal, Platform.Switch };
         }
+
+        // Return platform list not compatible with DXC (The list use by HDRP)
+        internal static Platform[] GetNeverUseDXCPlatformArray()
```

Comment:
This is required for now, but there is a PR coming which will prevent to require it